src/main/java/com/healthapp/project/Entity/Medication.java

```diff
@@ -4,7 +4,9 @@
 
 @AutoMatter
 public interface Medication {
-    int id();
-    String medication_name();
-    String strength();
+  int id();
+
+  String medication_name();
+
+  String strength();
 }
```

Comment:
I think i've mentioned this before but i'm using Automatter here. Extra long story short, it saves me a lot of lines by not having to generate getters/setters.. it's all in here. There's also a built in Builder class that's also pretty useful